nvtabular/inference/graph/ops/workflow.py

```diff
@@ -21,7 +21,7 @@
 from nvtabular.inference.triton.ensemble import _generate_nvtabular_config
 
 
-class WorkflowOp(InferenceOperator):
+class TransformWorkflow(InferenceOperator):
```

Comment:
Wonder if we should call this `WorkflowTransform`, `FeatureTransform`, or even just `Transform`

Reply:
`FeatureTransform` seems good